lib/internal/modules/cjs/loader.js

```diff
@@ -443,7 +443,6 @@ function trySelf(paths, exts, isMain, trailingSlash, request) {
   if (expansion) {
     // Use exports
     const fromExports = applyExports(basePath, expansion);
-    if (!fromExports) return false;
```

Comment:
@jkrems I've also removed this line as it never triggers.

Reply:
SGTM